libs-haskell/test-utils/DA/Test/Util.hs

```diff
@@ -77,15 +86,28 @@ withEnv vs m = bracket pushEnv popEnv (const m)
 
 newtype ShouldSucceed = ShouldSucceed Bool
 
-callProcessSilent, callProcessSilentError :: FilePath -> [String] -> IO ()
-callProcessSilent = callProcessSilent' (ShouldSucceed True)
-callProcessSilentError = callProcessSilent' (ShouldSucceed False)
+callProcessSilent :: FilePath -> [String] -> IO ()
+callProcessSilent cmd args =
+  void $ run (ShouldSucceed True) (proc cmd args)
+
+callProcessSilentError :: FilePath -> [String] -> IO ()
+callProcessSilentError cmd args =
+  void $ run (ShouldSucceed False) (proc cmd args)
+
+callProcessForStdout :: FilePath -> [String] -> IO String
```

Comment:
This is only needed for the `inspect-dar`. If you just read the DAR directly you should be able to undo all the changes here.

Reply:
It is often useful to be able to read the stdout during tests. So I would rather not undo the changes.

Lets address the `inspect-dar` improvement in a follow up PR. My preference would be to have a user facing command to get the package-id from a DAR and use that instead.

Reply:
Alright, let’s keep it for now so we can get this merged.